### ISSUE TITLE
Option to visualize unused space

### ISSUE THREAD

**Issue by pfdint** [shundhammer/qdirstat#173]
Is it possible to also visualize the empty space for a filesystem?
I think windirstat has this as a checkbox option. The empty space appears as its own contiguous box in the visualization.

**Comment by shundhammer** [comment]
Yes, WinDirStat has that, it's disabled by default, and there are good reasons for that: It's completely useless. It completely dwarfs everything else, making the treemap view pretty much unusable.

![windirstat-all-drives-with-free-space](https://cloud.githubusercontent.com/assets/11538225/20030681/3d30c662-a36a-11e6-862f-28e66c86160b.png)

See also https://github.com/shundhammer/qdirstat/issues/37 which this is a duplicate of.

**Comment by shundhammer** [comment]
Also, see the _Mounted Filesystems Window_ which is new since V1.7.

https://github.com/shundhammer/qdirstat/tree/master/screenshots#mounted-filesystems-window

**Comment by pfdint** [comment]
Wouldn't it only be useless if the disk is mostly empty? My disks are almost always brimming full, so I would find it useful to compare how much relative free space a deletion was buying me.
But, your choice seems made.
Thank you for your work on this project!

**Comment by shundhammer** [comment]
Even in the case where your disk is pretty full, that empty space would be by far the largest blob in the treemap; it would still dwarf everything else there. Even if the disk is 99% full, the remaining 1% of a modern disk is still _huge_ compared to any file on the disk, even if you keep virtual machine images and installation ISOs there.

**Comment by pfdint** [comment]
Is that because most files don't fill up a block? I thought qdirstat now reflects allocated size?
Oh you mean literally, if you have a 1TB disk, 1% of that is 10GB which would be a very obese file compared to the average file size of a typical system.
So the normal 90-95% occupancy I face would have a 50GB or 100GB tile, and the runner ups would by my occasional 10GB file along with the fleet of 5GB files I deal with. Which is okay for usability, and why I thought it was reasonable.
But if that 1TB assumption changes, especially as disk size continues to increase, 1% of a 4TB disk is now 40GB and 5% is now 200GB, which is now 20x the size of the runner up 10GB files I have. So the visualization's usefulness is already decreasing.
Since file sizes aren't expected to increase at the same rate as disk capacity (are they?), I understand now how this kind of feature would probably have diminishing usability into the future.

**Comment by shundhammer** [comment]
Right, that pretty much summarizes it.
